### ISSUE TITLE
Ember 2.0 support?

### ISSUE THREAD

**Issue by supersabillon** [Addepar/ember-table#400]
I'm using the ember-2.0 branch in my project, but trying to run the _Hello World table_ example gives me this error in the console:

`Uncaught Error: Assertion Failed: A helper named 'fixed-grid-layout' could not be found`

is this a known issue or am I doing something wrong?
thanks

**Comment by MartinMalinda** [comment]
The ember collection which is listed as dev dependency should probably be a dependency. Ember install ember-collection should solve this issue.

**Comment by cyril-sf** [comment]
@MartinMalinda is correct, I need to fix that.

**Comment by haldiggs** [comment]
will the 2.0 support ever be pushed out https://www.npmjs.com/package/ember-table only gets me the one supporting ember 1.12.x

**Comment by cwiese** [comment]
Has anyone stress tested this branch?  Once again we need a GRID in Ember land and there are not a lot of good options (like React land) .....I want to try Ember Table again

**Comment by DanChadwick** [comment]
It has pretty good performance with 1000-ish rows. Not sure about a huge number of rows. One limitation is that there are observers in the cells. These can be eliminated with a computed property if you can swallow some simplifications: a) constant contentPath for each column and b) cellContent is just a property (calling the model's getCellContent). There are probably lots of other opportunities for optimization too. The above only affects the visible cells, not the total number of cells.

Rows are created for each model. There is overhead there, obviously.

**Comment by cwiese** [comment]
Would this be the best choice for a Grid in the Ember-world today …with sortable, filterable, hide-able columns, that scrolls like Slick Grid :-)

**Comment by DanChadwick** [comment]
It has provision for doing sorting and filtering in that you can implement it yourself. It does not do it out-of-the-box. It does lazy rendering -- only what is in the viewport is in the DOM, so the total list size matters much less. There are still O(N) and probably at least O(N log N) performance because the whole list of rows is processed. Your performance mileage may vary.

**Comment by cfator** [comment]
When running against Ember/Ember-CLI: 2.4.5/2.4.3 with ember-collection: v1.0.0-alpha.5, I get this error when starting up Ember Server: Error: 

Assertion Failed: Using `{{view}}` or any path based on it ('arms-ui/templates/body-table-container.hbs' @ L5:C8) has been removed in Ember 2.0

Which is leading me to believe that ember-table is not ready for recent versions of Ember?

**Comment by MartinMalinda** [comment]
@cwiese I think when Glimmer 2 lands, current perf issues will be gone

**Comment by haldiggs** [comment]
is there any time frame for this? rough estimate is fine

On Mon, May 9, 2016 at 10:29 AM, Martin Malinda <notifications@github.com>
wrote:

> @cwiese <https://github.com/cwiese> I think when Glimmer 2 lands, current
> perf issues will be gone
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/Addepar/ember-table/issues/400#issuecomment-217898392>
>



-- 
Hal D. Diggs
<https://www.mcpvirtualbusinesscard.com/VBCServer/haldiggs/card> - "Non
sibi sed patriae" (Not for self, but for country)
Half of communications is listening, and you can't listen with your mouth.

**Comment by Datamance** [comment]
+1 to the comment from @cfator as I'm getting the exact same error

**Comment by Datamance** [comment]
@MartinMalinda or @cyril-sf - do you have an idea of when the 2.0-appropriate changes are going to be merged into the primary branch? Is there a new major release in the pipeline that will include Ember 2.0 support, or is this something that will be bundled with a more minor update?

**Comment by pzuraq** [comment]
Hey all, I'm closing this issue in favor of #418.

We are planning on releasing a new version soon, we've been hard at work on a next generation occlusion library, [vertical-collection](https://github.com/html-next/vertical-collection), to replace Ember.ListView. Now that it's nearly complete we're planning on working on the next generation.

We'd love your feedback, feel free to add any thoughts to the main discussion issue or ping us in #e-table on the Ember Community Slack!
